### DISCUSSION TITLE
Develop more educational notebooks for plasma science education

### DISCUSSION THREAD

**Discussion by pheuer** [PlasmaPy/PlasmaPy#2644] [Feature Ideas & Requests]
It would be great if PlasmaPy had more educational notebooks suitable to be incorporated in the curriculum of high school/undergrad/grad plasma physics classes. Let's brainstorm some ideas for possible notebooks?

**Comment by pheuer** [comment]
Ideas
- "What is a plasma": Saha equation, Paschen curve, number of particles per Debye sphere 
  - Saha equation and Debye length exists, Paschen curve needs to be added. 
- Debye shielding derivation: plot analytical potential solution and show how it agrees with the Debye length in PlasmaPy
  - Debye length is in PlasmaPy 

- "How is temperature defined?": Calculate temperature as the moment of the Maxwellian, show how it agrees with the normal temperature, then show how the 'temperature' changes with non-Maxwellian distributions 
  - No PlasmaPy functionality really necessary, but could use the existing thermal velocity function as part of this? Could optionally develop an arbitrary `DistributionFunction` class 

- Educational notebooks on MHD and two-fluid waves that summarize the different waves and plot their dispersion relations in a way where you can easily change the plasma parameters and see how the dispersion curves change. 

- Fusion reactivities and the Lawson parameter. Define the reactivity from the cross-section, then show the Lawson parameter calculated for different reactions to show why some fusion reactions are more accessible than others, 
  - Reactivities requires tabulated cross-sections and a nuclear reaction class using Particles to access them. The Lawson parameter involves also including other effects like radiation and conduction.

**Reply by cacsphysics** [reply]
I like the idea of having educational notebooks on MHD and two-fluid waves; this could be useful in many ways, and could be a good reference for everyone.
